python/xir/interfaces/strawberryfields_io.py

```diff
@@ -117,7 +117,7 @@ def to_xir(prog, **kwargs):
         else:
             if kwargs.get("add_decl", False):
                 if name not in [gdecl.name for gdecl in xir._declarations["gate"]]:
-                    gate_decl = GateDeclaration(name, len(cmd.op.p), len(wires))
+                    gate_decl = Declaration(name, [f"p{i}" for i, _ in enumerate(cmd.op.p)], wires, declaration_type="gate")
```

Comment:
No need to use `enumerate` when `range` will do:
```suggestion
                    params = [f"p{i}" for i in range(len(cmd.op.p))]
                    gate_decl = Declaration(name, params, wires, declaration_type="gate")
```

Reply:
That is true, although Codefactor has started complaining about this, recommending using `enumerate(...)` instead of `range(len(...))`, so I just started going with that instead. 😅

Reply:
It looks like [24 people on Stack Overflow](https://stackoverflow.com/a/11901308) agree!